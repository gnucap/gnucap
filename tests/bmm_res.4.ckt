# semiconductor "resistor" test
v1 1 0 dc 2 ac 5
r1 1 2 1k
r2 2 0 t1 (w=2u l=1u)
.model t1 r rsh=1k narrow=1u
.print op v(nodes) r(r*) iter(0)
.print ac v(nodes) 
.op
.ac
.list
.status notime
.end
