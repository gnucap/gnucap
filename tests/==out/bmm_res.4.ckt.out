# semiconductor "resistor" test 
r2: t1: effective length is negative or zero
#           v(1)       v(2)       r(r1)      r(r2)     
r2: t1: effective length is negative or zero
r2: short circuit
r2: short circuit
r2: short circuit
r2: short circuit
 27.        2.         20.n       1.K        10.u      
r2: t1: effective length is negative or zero
#Freq       v(1)       v(2)      
r2: short circuit
 0.         5.         50.n      
v1 ( 1 0 )  DC  2.  AC  5.
r1 ( 1 2 )  1.K
r2 ( 2 0 ) t1 l= 1.u w= 2.u
.model t1 r ( rsh=1k narrow=1u defw=NA( 1.u) tc1=NA( 0.) tc2=NA( 0.) tnom=NA( 27.))
